controller/label.go

```diff
@@ -149,3 +147,22 @@ func ConvertLabels(appl application.Application, request *http.Request, labels [
 	}
 	return ls
 }
+
+// ConvertLabelsSimple converts a array of Label IDs into a Generic Reletionship List
```

Comment:
s/a array/an array

Reply:
done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/6a0c0cfcaefc599fb01eb84241e7feb2381f5d13 thanks!